pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.86</version>
+        <version>5.3</version>
```

Comment:
shouldn't this PR be closed in favor of a new one with [the latest version](https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-5.4)? Dependabot is paused at this moment, so I guess that's the reason why this PR remains open.

Reply:
That's because to migrate to plugin-pom 5.0+ we need core 2.479.1+ and dependabot did not in the same pull request. Fixing it now.

Reply:
Done.